tools/mmp/mmp.csproj

```diff
@@ -46,172 +52,172 @@
     <Compile Include="Assembly.cs" />
     <Compile Include="error.cs" />
     <Compile Include="SdkVersions.cs" />
-    <Compile Include="..\..\external\mono\external\linker\src\tuner\Mono.Tuner\ApplyPreserveAttributeBase.cs">
+    <Compile Include="$(MONO_IOS_SDK_DESTDIR)\ios-sources\external\linker\src\tuner\Mono.Tuner\ApplyPreserveAttributeBase.cs">
```

Comment:
that does not work when opening the `Xamarin.Mac.sln` file inside VS4M

Reply:
maybe we can have a _static_ symlink that can be referenced in project files ?
and the destination can be updated whenever a new archive is downloaded

Reply:
yeah, any ideas how we can get it into there? the problem is that the file path changes depending on source-build or download-build.

One idea I had was to create a symlink at a known location that points to the correct place depending on which method you used to build, but not sure that's the right thing.

Reply:
I have an idea

Reply:
WIP, but something like this: https://github.com/rolfbjarne/xamarin-macios/commit/c669126f2fcabf78b8e363409f64d0ec404f3ae3

Reply:
Although now that I think about it, I like the symlink idea better

Reply:
does anyone know whether VSMac handles the symlink gracefully? i.e. it doesn't resolve and replace the reference with the target on save or similar shenigans.

the msbuild from rolf solution seems a little less fragile in that regard.

Reply:
Turns out VSMac handles the symlink perfectly so I went with that idea.